widget/ui/src/components/TokenAmount/TokenAmount.styles.ts

```diff
@@ -11,7 +11,19 @@ export const Container = styled('div', {
       },
       horizontal: { flexDirection: 'row', width: '100%' },
     },
+    centerAlign: {
```

Comment:
Why we do need this? it seems it doesn't do anything

Reply:
actually, it does but in the compoundVariants with a vertical direction due to another version of this component in Figma. But I changed it to make it more general.